common/util.go

```diff
@@ -336,6 +341,10 @@ func IsServiceClientTransientError(err error) bool {
 }
 
 func IsServiceHandlerRetryableError(err error) bool {
```

Comment:
Will this disable all retries, or some retries like between FE and History will still happen?